### ISSUE TITLE
Treesitter support for language parsing

### ISSUE THREAD

**Issue by gamecreature** [edbee/edbee-lib#78]
https://youtu.be/Jes3bD6P0To.

https://github.com/tree-sitter/tree-sitter

**Comment by brupelo** [comment]
Bookmarked both links, thanks to share!

**Comment by brupelo** [comment]
Already watched the video... after spending the last weeks getting familiar with almost all existing textmate derivative work living in github I wonder now... is there any good reason of considering textmate syntax&highlighting instead tree-sitter approach? And the only reason that comes to my mind is just one:

There are lots of existing textmate bundles for almost all languages ready to consume... can you think of any other advantages of textmate vs treesitter in terms of expected quality?

**Comment by gamecreature** [comment]
Indeed the last one is a good reason...
Changing the code to tree-sitter, isn't very trival either.
With tree-sitter you are limited to the parsers writter for it

**Comment by brupelo** [comment]
Yeah... Plus, writting new tree-sitter grammars wouldn't be a 5min job neither. Now... I wonder, the way the author sold wonderfully his product on the video looked like textmate syntax being really limited but i wonder, is that really so? I mean, wouldn't some of the presented textmate problems in the video solvable somehow? Or are they really true limitations inherent to the fact of using regular expressions?
